frontend/packages/network-attachment-definition-plugin/integration-tests/views/nad.form.view.ts

```diff
@@ -0,0 +1,13 @@
+import { $ } from 'protractor';
+
+export const closeForm = $('#cancel');
+export const formHeader = $('co-m-pane__heading'); // TODO Find more specific selector
```

Comment:
There is missing `.` in the selector, element with this selector is actually never present.
Should be `$('.co-m-pane__heading');`

Reply:
Done!